deployer/cluster.py

```diff
@@ -61,6 +61,18 @@ def deploy_support(self, cert_manager_version):
         )
         print_colour("Done!")
 
+        # FIXME: Revert this addition after one upgrade is made
+        print_colour("Fixing prometheus v22 migration...")
+        subprocess.check_call(
+            [
+                "kubectl",
+                "delete",
+                "--namespace=support",
+                "deploy/support-prometheus-server",
+            ]
+        )
+        print_colour("Done!")
+
```

Comment:
The fix idea is to delete the deployment first, and then let the helm chart install add it back. Like that, no change request will need to be made as part of removing the matchSelector label which isn't allowed because that section is immutable.